css/properties/transition-timing-function.json

```diff
@@ -0,0 +1,167 @@
+{
+  "css": {
+    "properties": {
+      "transition-timing-function": {
```

Comment:
The comments about transition-delay.json are also valid here. Please apply them here too.